Ruby-Project/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+end
```

Comment:
- [ ] kindly add the method `add_rental` to this class the method should take parameter `person, date` and create a new instance of Rental class